Tests/LibWeb/Text/input/DOM/Element-text-transform-fullwidth.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8"/>
+<style type='text/css'>
+.test { text-transform: full-width; }
+</style>
+</head>
+
+<script src="../include.js"></script>
+<body>
+<div class="test">
+    <div>[<span>　</span>]<br />[<span> </span>]</div>
```

Comment:
Can we add some readable text to this test as well? ASCII alpha characters can also get the fullwidth treatment

Reply:
I copied the rest of the test from WPT